src/core/cubic.c

```diff
@@ -767,6 +897,11 @@ CubicCongestionControlInitialize(
     Cubic->InitialWindowPackets = Settings->InitialWindowPackets;
     Cubic->CongestionWindow = DatagramPayloadLength * Cubic->InitialWindowPackets;
     Cubic->BytesInFlightMax = Cubic->CongestionWindow / 2;
+    Cubic->MinRttInCurrentRound = UINT32_MAX;
+    Cubic->MinRttInLastRound = UINT32_MAX;
+    Cubic->HyStartAckCount = 0;
+    Cubic->HyStartRoundEnd = Connection->Send.NextPacketNumber;
+    CubicCongestionHyStartResetPerRttRound(Cubic);
```

Comment:
Is this call redundant? You are setting all that values that get reset in CubicCongestionHyStartResetPerRttRound right above the call/

Reply:
same in CubicCongestionControlReset